notebooks/Emukit-task-spread-of-disease-with-seir-model.ipynb

```diff
@@ -0,0 +1,908 @@
3bR1NsWe8MVPlWxMVdA1cU+Oho2nODCf3elkKxa4Qt3QMXbsldg/g3Ghqw7xqgeDBo2n8IOxb1RqiaviJoTi2qXGj3KbS6m78RbuIaxqEUvLslQKVdn/WNpSweWe+znRlfatH2jWZqXT2lQ9doVDcPUxXWsS3OX13TWEthHhECPEI8KQQ4h8LIT6yfN/S/YoDQsPz+YOLMzQ8n7hlMJp2ooi9VZq6FIo7ESEEh1L2bUJtNOVgGxq9DTzUuZjNkYxDq89NdVHOdZy4pTNT7VBoeH0V7zvFMXWEEDx7pUBxE5F5G6EhyLgmz1wprNs4KoTA1DUmisprrVAodpeG59PsBljGlu3VAKynrH7hPX+/eVa6BD61rSMqdp2JQpOvvb2IEFHE3ulDST4wlkEI1bSouHsYTti3jc3WNY37D6d4bapCPuGs+/X3H04zV50nlCGa2N4JdzX0pQmN05U2C3UPP5CMpOx98/60DY2uD89dL/GhUzmGN3ieNsIxDZpeh/MTZb79zPCaj0s5BvO1DsdyLkmVs69QKHaJxXoHbQfn3zWFtZTyk9v+rop9Q9cP+S9vzFFodIlbOrmYxSPHMwwlVcSe4u4i4USnOynlLaL1ZD7OqzcqSCSCtU+mccvg9KEEV+YbJJz+CWuIKrRHMy5G3aPU7OKHIUcy7o5O7v3EMjQI4PlrRR4/mWMktbOm52zC5upCk+MZlxP5xKqPEUJgGxoTxSYPHs3s6HgKhUKxGcJQMl3ukLQN6tu0ovX300Gx75ivdfjihRkAko7B8VyMR45nMXX10ivuLkxdIxszb/NUpx2T0ZRDcxN51aeHExi6ht+n+L2bEUIwkrQ5lLSpdXwmS63bPOF7iaVr2IbOi9dLzGzgkd4IDUE2ZvLNa8V1G4SSjslC3aO6zpRMhUKh6Be1Tg8/DDF2oJGUurrDeebyIpOlFo6pkXQMHj2R5Wg2ttfLUij2hNFVYvcAzo0mNtUBbuk6DxxJ0u5j/N7NCCHIJ2yOpB1a3YCJYnND//duYuoajqVzfqLMZHlnqR22oSMlvHCtiFwn8tA1DcYLKiFEoVAMntlqB2uHhUclrO9gglDyH85PEUpI2gbHcjE+eCJL3FZNi4q7k5RrrXr/kUwMUxf44cYi9lg2Rtrtb/zee8nELI5nY3hByPVCc8Pkkt3E0DRils4rkxXGi40dfa9s3GKi1OLK/NrfJ2EbFJvdfZeaolAo7ix6QchCvbNjjbTmVwsh/ux6Xyil/N0dHVkxcBqez+szbSxdEHcMHj2e4cwhFbGnuHtxLR3H1OkF4S12KFPXODuS5O3ZGkMbNOdpQuPhYxn++NIioaENzAedcAxODsW5UWoxXmhyNOOScvdHE5+uacRtuHCjSiDhnnx8XX/6WggEQ3GLF64XGUk7pNZoUoxbOlcXGzxyPLtvmjoVCsWdRbnZJQzZ8Tl9vYr19y3d/grRkJgfWro9AXx2R0dVDJym5/PcTI9eIIlbOmNph0dO5Ejvkw9mhWKviPzUt9s+7hlO4G/S3ZFxLe4ZjtMacM6ya+rck4/jmDpTlTYL9c6+iePThUbC0Xl9qsrl+fq6do71sHQdQ9N47kqBYI2fLWYZ1No9yi3ltVYoFINhutImZu18aN6awlpK+Tkp5eeIovUekFL+OSnlnwPet+OjKgbO9cUGT01LdE0Qsw0+cDzDudHUXi9LodhzsnFrVQGXjVnkE9amp22dHU1iGTrdAXugl6c0ZlyTQqPLVKV92xTJvUJbEtdvzdV5Z2774joTs5irdXh7rrbmY+KWybWFxr65sFAoFHcO7W5Atd0jts2hMDezGY/1SSnl7E1/nwfu2/GRFQOj64f8h2/dYLYFMVPnSMblkeM5MjFVrVYoEraBJgThKgLt/sOpTU/7MzWdh4+l6QyokfFmNCE4nHYYSdrUOz7Xi026A/R4bwVNaCRsnXfm6rw5W9u2uB5K2Lw8Uaa8hpfatXTqXZ9Cw9vJchUKheI2ik1vG2a21dmMsP6GEOKPhBA/IoT4S8AfAE/26fiKATBf6/CVN+ZwdUnS0XnoaJr3j6WVN1GhIBrIkk9atLq3NwQezbjoGgSbaGIEGEk5HE73fyLjagghGErYHM/G6AUh14vNfTPyO6pcG1yZb/D6dHVb4trQoji/Z68U1mwiTdoGVxYa+yqGUKFQHGyklEyV2yTs/hQfNxTWUsqfAH4VeBj4APBrUsqf7MvRFX0nDCV/cHGGQqPLB/PRB/8jJ7IMxdVAGIVimUNJh84qYtg2dM6MJKlsMjdZIHjwaBoBBHJ3KsgJx+BUPo4uBBOlFov1/TEGXROChGNwrdDk4lSVcBviOuWYFJtd3phZ3RJiGzqeHzJX3fl4dYVCoQCoez7tHYwwfy+b/S4vA38gpfw88EdCCBUtsU8pt7r89vkpAB7NS+4bSfLo8SyapqrVCsUySWdtH92ZfAJ/CxXRmGlw/+EULW/wlpBlbCNqakw7JosNj8lSayBDa7aKJgRJ22Ci2OTVyQrhNi428gmLVycrLDRWF88px+RaobFvrDAKheJgs1jz+jo0b8PvJIT4H4EvAP9i6a6jwO/1bQWKvvLUpUXGi01sQ5C2BY+fyjGcWj8+TKG427ANnZRjrlq1ziUssq5Jq7d5m8XJocFnW78XTRMcySxZUboB1wr7wxoihCBhG0yVW7w8Wd5yJV9fspU8/U4Bz7/99TF1jSCUTO9w+qNCoVAEoWS22ibRx/kem5Hofx34GFADkFJeBg71bQWKvlHr9Ph/vzmOlJBwTGxD47GTOXRVrVYobmMkba86eEUgeN+RNPXO5kXqcra1H4SrNkUOCiEE2ZjFqXwcbdka0vDYa2fIsrieqXQ4P7F1cR23DNo9n/Pj5VX/PeNajBdbq14YKRQKxWaptqNY4n7qpM0Ia09KudKmLYQwYJtt34qB8tJ4mdemq1i6IBczidsGhzPuXi9LodiXZGLWmrF1YzkXXWzNN72cbb0XVWPH1Dm
```

Comment:
`stopping_condition_emukit` isn't used and can be removed

Reply:
removed